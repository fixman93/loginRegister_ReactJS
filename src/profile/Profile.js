import React, { Component } from 'react'
import { Redirect, Link } from 'react-router-dom'
class Profile extends Component {
  render() {
    let token = localStorage.getItem('token')
    console.log('token', token)
    if (!token) {
      return <Redirect to='/' />
    }
    else {
      return (
        <div>
          Profile<br />
          <Link to='/'>Login/Register Page</Link>
        </div>
      )
    }
  }
}

export default Profile
