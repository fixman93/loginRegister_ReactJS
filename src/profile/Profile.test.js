import React from 'react'
import Adapter from 'enzyme-adapter-react-16'

import { shallow, configure } from 'enzyme'
import Profile from './profile'
configure({ adapter: new Adapter() })

describe('render profile', () => {
  const profile = shallow(<Profile />)
  it('snapshot profile page', () => {
    expect(profile).toMatchSnapshot()
  })
})