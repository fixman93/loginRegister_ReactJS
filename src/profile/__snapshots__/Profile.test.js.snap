// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render profile snapshot profile page 1`] = `ShallowWrapper {}`;
